### PR TITLE
reload the page when native gets a `back` call

### PR DESCRIPTION
**Pull request by liulianci**
added a flag `Refresh` to check refreshing the page or not. the default behavior is not refreshing the page.
This is an example of `Back` NavRouteObject: ["path": path, "jsonPayload": payload, "refresh": true
], jsonPayload will be added as an extra initialProperties when you reload the page

### CODE REVIEW

File: ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift [hunk 1]
```diff
@@ -53,4 +57,8 @@ import UIKit
         }
         return nil
     }
-}
+
+    func reloadeView(viewController: UIViewController) {
+        self.loadView(viewController: viewController)
+    }
+}
```
Comment (friederbluemle) [inline ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift]: Don't remove the newline character on the last line please ;)

Comment (jw2175) [reply ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift]: you have an extra 'e' in method name (reloadeView)

File: ios/ElectrodeApiImpl/APIImpls/Navigation/ENNavigationAPIImpl.swift [hunk 2]
```diff
@@ -58,7 +58,9 @@ class ENNavigationAPIImpl: NSObject {
             let d = data as? ErnNavRoute
             let ernData = d?.toDictionary() as? [AnyHashable : Any]
             let path = ernData?["path"] as? String ?? nil
-            self.delegate?.popToViewControllerWithPath(path: path, completion: { (message) in
+            let refresh = ernData?["refresh"] as? Bool ?? false
+            let jsonPayload = ernData?["jsonPayload"] as? String ?? nil
```
Comment (jw2175) [inline ios/ElectrodeApiImpl/APIImpls/Navigation/ENNavigationAPIImpl.swift]: do we still need these lines?

Comment (liulianci) [reply ios/ElectrodeApiImpl/APIImpls/Navigation/ENNavigationAPIImpl.swift]: 👍

**Pull request merged**